src/gamestates/loadGameMenu.lua

```diff
@@ -42,8 +42,7 @@ function LoadGameMenu.keypressed(key)
 end
 
 function LoadGameMenu.mousepressed(x, y, mouseButton)
-	local loadGameChoice = LoadGameMenu.menu:pressed(x, y)
-	LoadGameMenu.LoadGame(loadGameChoice)
+	LoadGameMenu.LoadGame(LoadGameMenu.menu:getButtonAt(x, y))
```

Comment:
It's unfortunate that when I made this change I didn't make it in a backwards compatible way.